@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -871,7 +872,7 @@ export class TextElementsRenderer {
                 ++this.m_initializedTextElementCount;
             }
         }
-        return textElement.loadingState === LoadingState.Initialized;
+        return textElement.glyphs !== undefined;
```

Comment:
why has this changed?

Reply:
When a label is replaced, the new label must inherit temporally the glyphs of the old label while its own glyphs are loaded asynchronously. Otherwise the replacement won't be rendered in the meantime. Nevertheless, `initializeGlyphs` will load the final glyphs for a label replacement and replace the inherited ones as soon as they're loaded.
So `initializeGlyphs` must return true as soon as a label has some glyphs assigned (either the final ones or the temporally inherited).

Reply:
I'll document this in the code.

Reply:
There some comment in `TextElement.replace()`, but I'll add more in `initializeGlyphs`